src/main/java/com/gregtechceu/gtceu/integration/kjs/builders/block/CoilBlockBuilder.java

```diff
@@ -25,7 +24,7 @@ public class CoilBlockBuilder extends BlockBuilder {
     @Setter
     public transient int temperature = 0, level = 0, energyDiscount = 1, tier = 0;
     @NotNull
-    public transient Supplier<@Nullable Material> material = () -> null;
+    public transient Supplier<Material> material = () -> null;
```

Comment:
... you forgot to change the default supplier
```suggestion
    public transient Supplier<Material> material = () -> GTMaterials.NULL;
```